Podfile.lock

```diff
@@ -1,16 +1,16 @@
 PODS:
-  - JWPlayer-SDK (3.13.0)
+  - JWPlayer-SDK (3.16.2)
```

Comment:
I don't think this file should be checked in at all. This is generated dynamically by the cocopods.

Reply:
Added `Podfile.lock` to the gitignore.

Reply:
I also made sure `Pods/` was not commented-out in the gitignore.